redecs_agent/redecs_agent.go

```diff
@@ -44,10 +133,24 @@ func logErrorNoFatal(err error) {
 var dockerClient *docker.Client
 var redisClient *redis.Client
 
-func reportServiceActive(serviceName string) {
+func reportServiceActive(serviceName string) error {
```

Comment:
Add one line comments for the long functions in the format `// reportServiceActive reports this service as active in redis`